### ISSUE TITLE
Unable to compile CouchDB with dunfell branch

### ISSUE THREAD

**Issue by dmrayt** [meta-erlang/meta-erlang#166]
I got an error while trying to compile couchdb: "No package 'erl_ei' found"

I'm using dunfell branch and erlang version 20.3.8.26.

**Comment by joaohf** [comment]
Right, that happens for 20.x, 21.x and 22.x because the patches for pkg-config (from this layer) were not available (yet) for these erlang versions. Also the couchdb 3.1.2 does not support erlang 23.x or 24.x.

So, two options here: bump couchdb to 3.2.x or backport the pkg-config patch to the missing erlang versions.

Does updating couchdb work for you ?

**Comment by dmrayt** [comment]
The patch has already work before. I was using CouchDB 3.1.1 with erlang version 20.3.8.26 (commit 286eb34 from March 2021)

If you can add 3.2.x to dunfell, I will try if it works.

**Comment by joaohf** [comment]
Hello @dmrayt 

I've added again the 3.1.1 and 3.1.2 now using apache archives that is suppose to not disappear. Please, try it using the Erlang 22.3.4.22 or 21.3.8.24.

Unfortunately the Erlang version 20.3.8.26 is not building yet.

The couchdb 3.2.x require some dependencies that Yocto/OE dunfell branch does not have. So, it's better to stick with 3.1.x on dunfell.

**Comment by dmrayt** [comment]
I was able to build couchdb 3.1.2 and erlang 20.3.8.26 with the current commit in dunfell branch.
Thanks.
